src/main/kotlin/de/uniwuerzburg/omod/core/MiscModels.kt

```diff
@@ -113,5 +113,7 @@ data class Activity (
     val stayTime: Double?,
     val location: LocationOption,
     val lat: Double,
-    val lon: Double
+    val lon: Double,
+    val cellId: Int
```

Comment:
I don't like this. I think adding a TAZ file is a niche usecase that shouldn't change the API.